ai_summary/indico_ai_summary/controllers.py

```diff
@@ -0,0 +1,64 @@
+# This file is part of the Indico plugins.
+# Copyright (C) 2002 - 2025 CERN
+#
+# The Indico plugins are free software; you can redistribute
+# them and/or modify them under the terms of the MIT License;
+# see the LICENSE file for more details.
+
+import os
+import time
+
+from flask import jsonify, request
+
+from indico.modules.events.controllers.base import RHEventBase
+from indico.modules.events.notes.util import get_scheduled_notes
+
+from indico_ai_summary.utils import cern_qwen, chunk_text, html_to_markdown, markdown_to_html
+
+
+# summarize the event
+class SummarizeEvent(RHEventBase):
```

Comment:
I believe this should inherit from `RHManageNoteBase` or similar which ensures that the user can manage the minutes.